app/views/layouts/_sidebar.html.erb

```diff
@@ -3,58 +3,58 @@
           <%= menu_item(label: cases_index_title, path: casa_cases_path) %>
           <% if current_user.volunteer? %>
-            <%= menu_item(label: "Case Contacts", path: case_contacts_path) %>
+            <%= menu_item(label: t(".menu_item.case_contacts"), path: case_contacts_path) %>
           <% end %>
           <% if current_user.serving_transition_aged_youth? %>
-            <%= menu_item(label: "Emancipation Checklist(s)", path: emancipation_checklists_path) %>
+            <%= menu_item(label: t(".menu_item.emancipation_checklists"), path: emancipation_checklists_path) %>
           <% end %>
-          <%= menu_item(label: "Admins", path: casa_admins_path, visible: policy(CasaAdmin).index?) %>
+          <%= menu_item(label: t(".menu_item.admins"), path: casa_admins_path, visible: policy(CasaAdmin).index?) %>
         <% end %>
       </div>
     </div>
     <div class="sidebar-footer">
       <div class="list-group list-group-flush account-details">
         <li class="list-group-item">
-          <span class="label">Location</span>
+          <span class="label"><%= t(".footer.location") %></span>
           <span class="value"><%= page_header %></span>
         </li>
         <li class="list-group-item">
-          <span class="label">Role</span>
+          <span class="label"><%= t(".footer.role") %></span>
           <span class="value"><%= current_role %></span>
         </li>
       </div>
       <div class="list-group list-group-flush footer-nav">
         <% if user_signed_in? %>
-          <%= link_to "Edit Profile", edit_users_path, class: "list-group-item" %>
+          <%= link_to t(".link.edit_profile"), edit_users_path, class: "list-group-item" %>
           <% if policy(:application).see_court_reports_page? %>
-            <%= link_to "Generate Court Reports", case_court_reports_path, class: "list-group-item" %>
+            <%= link_to t(".link.generate_court_reports"), case_court_reports_path, class: "list-group-item" %>
           <% end %>
           <% if policy(:application).see_reports_page? %>
-            <%= link_to "Generate Reports", reports_path, class: "list-group-item" %>
+            <%= link_to t(".link.generate_reports"), reports_path, class: "list-group-item" %>
           <% end %>
           <% if policy(:application).see_import_page? %>
-            <%= link_to "System Imports", imports_path, class: "list-group-item" %>
+            <%= link_to t(".link.system_imports"), imports_path, class: "list-group-item" %>
           <% end %>
           <% if policy(:application).modify_organization? %>
-            <%= link_to "Edit Organization", edit_casa_org_path(current_organization), class: "list-group-item" %>
+            <%= link_to t(".link.edit_organization"), edit_casa_org_path(current_organization), class: "list-group-item" %>
           <% end %>
-          <%= link_to "Report a site issue",
+          <%= link_to t(".link.report_site_issue"),
```

Comment:
Failing test
```
  1) layout/footer when not logged in renders report issue link on the footer
     Failure/Error: expect(rendered).to have_link("Report a site issue", href: "https://rubyforgood.typeform.com/to/iXY4BubB")
       expected to find link "Report a site issue" with href "https://rubyforgood.typeform.com/to/iXY4BubB" but there were no matches
     # ./spec/views/layouts/footer.html.erb_spec.rb:7:in `block (3 levels) in <top (required)>'
```

Reply:
The root key should be `footers` instead of `footer` :D I've just fixed it